### ISSUE TITLE
Error with installation command

### ISSUE THREAD

**Issue by rnollet** [jech/storrent#1]
## Summary of the problem

The installation command given in the [README](https://github.com/jech/storrent/blob/master/README) is `go install github.com/jech/storrent@latest`. That command gives me an error and does not seem to compile or install `storrent`.

## My configuration

- Debian GNU/Linux 11 (bullseye)
- go version go1.15.9 linux/amd64.

## Details of what I tried

When I execute `go install github.com/jech/storrent@latest` outside of the `storrent/` source directory, I get the following error:
```
package github.com/jech/storrent@latest: cannot use path@version syntax in GOPATH mode
```
When I do it at the root of the `storrent/` source directory, I get the following error:
```
package github.com/jech/storrent@latest: can only use path@version syntax with 'go get'
```
When, in the same `storrent/` directory, I do instead `go get github.com/jech/storrent@latest`, I get again:
```
go: cannot use path@version syntax in GOPATH mode
```

## Solution used

In the end, in the `storrent/` directory, I did `go build` and it seemed to compile `storrent` correctly.

## Question

What am I doing wrong? Do I miss some configuration? Some preliminary command?

**Comment by jech** [comment]
Fixed in 5df64f81286f2c3804bd1d0cedf1640fd999e883.  Thanks.

**Comment by rnollet** [comment]
Works perfectly. Thanks. :kissing_heart:
